File: benchmarks/cross-lang/fib40/go/main.go
package main

import "os"

func fib(n int64) int64 {
	if n < 2 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	r := fib(40)
	os.Exit(int(r & 255))
}

File: benchmarks/cross-lang/fib40/c/main.c
#include <stdint.h>

static int64_t fib(int64_t n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

int main(void) {
    int64_t r = fib(40);
    return (int)(r & 255);
}
